test/specs/conditions/preflight_number.yaml

```diff
@@ -0,0 +1,5 @@
+condition:
+  preflight.x-version>: 2
+
+vcl: 'std.atoi(req.http.x-preflight-x-version) > 2'
```

Comment:
this implies numerical versions. how would we handle semvers? `4.3.1` or custom versions `breaking-september < breaking-ocotober`, or release names: `antbear < bandicoot < coati` ?

Reply:
For our use case, we'd have the preflight service return a month name and use `preflight.x-version=: september` in the condition.